api/krusty/helmchartinflationgenerator_test.go

```diff
@@ -233,3 +236,66 @@ spec:
   type: ClusterIP
 `)
 }
+
+func TestHelmChartInflationGeneratorMultipleValuesFiles(t *testing.T) {
```

Comment:
This PR is also adding support for `APIVersions`, `Description`, `NameTemplate` and `SkipTests`. Can we please get coverage for those too? I would expect to be able to test anything we're introducing flags for, since if it doesn't affect the output then the flag presumably wouldn't be relevant to us?

Reply:
I added tests for `apiVersions, skipTests, and nameTemplate`. Couldn't see any output difference with `Description` so I removed it.